dymos/test/test_load_case.py

```diff
@@ -449,20 +221,14 @@ def test_load_case_lgl_to_rk4(self):
         system_cases = cr.list_cases('root')
         case = cr.get_case(system_cases[-1])
 
-        # Now change the grid so that the previous case won't have the same number of variables
-        phase.options['transcription'] = dm.RungeKutta(num_segments=50)
-
-        # Initialize the system with values from the case.
-        # We unnecessarily call setup again just to make sure we obliterate the previous solution
-        # First reset the connections at the top level model until fixed in OpenMDAO
-        p.setup()
+        # create a problem with a different transcription with a different number of variables
+        q = setup_problem(dm.RungeKutta(num_segments=50))
 
         # Load the values from the previous solution
-        dm.load_case(p, case)
+        dm.load_case(q, case)
 
         # Run the model to ensure we find the same output values as those that we recorded
-        fail_flag = p.run_driver()
-
+        fail_flag = q.run_driver()
         self.assertFalse(fail_flag)
```

Comment:
We're testing to verify that we get the same solution (and I realize it was previous this way as well), but it's probably more correct to also test at least the initial/final state and control values, as well as the t_duration value between the two problem objects to verify that they match after a q.run_model, not q.run_problem.

Reply:
I'm not sure exactly what you mean, but I added a check to make sure the control curves match like the other tests.